.markdownlint-cli2.jsonc

```
@@ -79,7 +79,9 @@
     // link-fragments
     // Disabled as it uses a different heading conversion algo to the TOP website
     "MD051": false
-  },
+    },
+    // Custom rules specific to the project
+    // Docs for each rule can be found in `./markdownlint/docs`
```

I meant that the comments were indented too much, not that the closing brace wasn't indented enough.
```suggestion
  },
  // Custom rules specific to the project
  // Docs for each rule can be found in `./markdownlint/docs`
```